core/sequencer/server.go

```diff
@@ -248,6 +249,7 @@ func (s *Server) DefineRevisions(ctx context.Context,
 }
 
 // readMessages returns the full set of EntryUpdates defined by sources.
+
```

Comment:
I'd remove the blank line.

Reply:
Yep, not sure how that made it in there.